mmv1/products/spanner/Database.yaml

```diff
@@ -155,3 +155,10 @@ properties:
       - :GOOGLE_STANDARD_SQL
       - :POSTGRESQL
     default_from_api: true
+  - !ruby/object:Api::Type::Boolean
+    name: 'enableDropProtection'
```

Comment:
Can we adda a test on this field. Plus, in the description, it says "Defaults to false" . Can you confirm if that's true and add default value for the field. Thanks.

Reply:
Done